client/core/core.go

```diff
@@ -4853,6 +4904,25 @@ func (c *Core) PreOrder(form *TradeForm) (*OrderEstimate, error) {
 
 // Trade is used to place a market or limit order.
 func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
+	resultChan, err := c.trade(pw, form)
+	if err != nil {
+		return nil, err
+	}
+	submissionResult := <-resultChan
+	if submissionResult.err != nil {
+		return nil, submissionResult.err
+	}
+
+	return submissionResult.ord, nil
+}
```

Comment:
I'm not following. From the caller's perspective, it would seem that the `Queue bool` has no effect whatsoever.

Reply:
Yes, I think the PR is WIP at the moment.  The diff suggested in https://github.com/decred/dcrdex/compare/master...buck54321:async-login?expand=1 is not compatible with the modified TradeForm